netneurotools/plotting.py

```diff
@@ -761,18 +764,22 @@ def plot_point_brain(data, coords, views=None, cbar=False, figsize=(4, 4.8),
                   axial=(90, 180), ax=(90, 180),
                   coronal=(0, 90), cor=(0, 90))
 
-    # coordinate space needs to be centered around zero for aspect ratio
-    coords = zscore(coords)
     x, y, z = coords[:, 0], coords[:, 1], coords[:, 2]
+
     if views is None:
         views = [_views[f] for f in ['sagittal', 'axial']]
     else:
         if not isinstance(views, Iterable) or isinstance(views, str):
             views = [views]
         views = [_views[f] for f in views]
 
+    if views_orientation == 'vertical':
+        ncols, nrows = 1, len(views)
+    elif views_orientation == 'horizontal':
+        ncols, nrows = len(views), 1
```

Comment:
Do you think it would make sense to also change (switch x/y) the figsize if not set explicitly by the user?

Reply:
Hum, I am wondering if its ok to have the default parameter being changed without the user being explicitly told about the change. I.E. currently, we set the default figsize to (4, 4.8). If we then change it to (4.8, 4), it might be confusing for the user. What I think could be a good idea is to set this default parameter to None, and then automatically adjust the figsize depending on a) the number of views the user wants, and the orientation of the views. For example, if the user wants two views, and horizontal, the figsize would be set to (9.6, 4).

Reply:
Good idea, I agree!

Reply:
I replaced the figsize parameter by the views_size parameter (so as not to confuse the user). The actual figsize parameter is adjusted according to the number of views and the orientation of the different views.